stdlib/testing/testdata/totime.flux

```diff
@@ -29,8 +29,7 @@ outData = "
 ,,0,2018-05-22T19:52:00Z,2030-01-01T00:00:00Z,k,m,2018-05-22T19:00:00Z
 "
 
-t_toTime = (table=<-) =>
-	(table
+t_toTime = (table=<-) => table
```

Comment:
So I guess the parser is caught the unbalanced `(`?

Reply:
It took the rest of the expression as part of the one in the parenthesis and I noticed that it changed the error structure. Then I wondered why there was an error to begin with and I found this.